packages/platform/atoms/hooks/useAvailableSlots.ts

```diff
@@ -15,7 +15,7 @@ export const QUERY_KEY = "get-available-slots";
 export const useAvailableSlots = ({
   enabled,
   ...rest
-}: GetAvailableSlotsInput_2024_04_15 & { enabled: boolean; isTeamEvent?: boolean; teamId?: number }) => {
+}: GetAvailableSlotsInput_2024_04_15 & { enabled: boolean }) => {
```

Comment:
isTeamEvent and teamId types are now defined in GetAvailableSlotsInput_2024_04_15